worker-message-prioritization-target-capacity-calculators/src/main/java/com/github/workerframework/workermessageprioritization/targetqueue/K8sTargetQueueSettingsProvider.java

```diff
@@ -200,7 +200,7 @@ private static long getLabelOrDefault(
         final long defaultValue)
     {
         if (!labels.containsKey(labelName)) {
-            LOGGER.error(
+            LOGGER.warn(
```

Comment:
As the default log level is info these would still happen, so for this one put it to debug